packages/liveblocks-zustand/src/index.ts

```diff
@@ -372,7 +372,7 @@ function updatePresence<
 ) {
   for (const key in presenceMapping) {
     if (typeof newState[key] === "function") {
-      throw mappingToFunctionIsNotAllowed("value");
+      throw mappingToFunctionIsNotAllowed(key);
```

Comment:
@GuillaumeSalles You authored this 7 months ago. Not sure if there is a subtle reason for this that I'm overlooking, but if I had to guess this was a TDD'ed version that wasn't not generalized after getting to green?

Reply:
Going to assume this is the case, and including this with 0.18.3. Lmk if I happen to have overlooked something after all.